src/modules/test/test.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { TestRepository } from './test.repository';
+import { CreateTestEntityDto } from './dto';
+import { TestEntity } from './test.entity';
+
+@Injectable()
+export class TestService {
+  private readonly testRepository: TestRepository;
+
+  constructor(testRepository: TestRepository) {
+    this.testRepository = testRepository;
+  }
+
+  // Returning all data
```

Comment:
I think its obvious from the name what function doing, or the naming should be better)... remove comments